src/js/view/controls/next-display-icon.js

```diff
@@ -1,11 +1,16 @@
 import displayIconTemplate from 'view/controls/templates/display-icon';
 import NEXT_ICON from 'assets/SVG/next.svg';
 import utils from 'utils/helpers';
+import UI from 'utils/ui';
 
 export default class NextDisplayIcon {
-    constructor(model) {
+    constructor(model, api) {
         const element = utils.createElement(displayIconTemplate('next', model.get('localization').next, NEXT_ICON));
 
+        new UI(element).on('click tap', function() {
+            api.next();
```

Comment:
Also, I noticed that `this.iconUI = ...` was removed. Is it no longer needed anywhere? If so, can we get rid of it from the rewind display icon as well?